### PR TITLE
Include Information about needed dependency

### PR DESCRIPTION
**Pull request by nilsmartel**
I vividly remember that these details used to mean game over for me, when trying out new libraries some years ago.

These things are often undocumented, and I think it's worth putting this up front in the README.

If I find the time, I will see that these instructions are complete (I already have some libraries for development installed, might have missed something others have not installed), and also on pair for ubuntu/pop_os etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,9 +72,24 @@ By default, view nodes are strongly typed. The type of a container includes the
 
 The type erasure of View nodes is not an easy trick, as the trait has two associated types and the `rebuild` method takes the previous view as a `&Self` typed parameter. Nonetheless, it is possible. (As far as I know, Olivier Faure was the first to demonstrate this technique, in [Panoramix], but I'm happy to be further enlightened)
 
+## Prerequisites
+
```
Comment (mriise) [inline README.md:76]:
```suggestion

## Git deps 
```sh
cargo update
# at the moment dependencies like glazier are through git and need to be pulled in manually

```

File: README.md [hunk 2]
```diff
@@ -72,9 +72,24 @@ By default, view nodes are strongly typed. The type of a container includes the
...
+In order to run xilem on linux systems, the development library [xkbcommon] needs to be installed.
+
+On Fedora this can be installed using:
+
+```sh
+sudo dnf install libxkbcommon-x11-devel.x86_64
+# i686 is a less common architecure, but if you have it, use libxkbcommon-x11-devel.i686 instead
+```
+
+
```
Comment (mriise) [inline README.md:88]: forgot to add debian dep, not sure if i686 applies here as well?

```suggestion
Debian:

```sh
sudo apt-get install libxkbcommon-x11-dev
```
```

**Comment by mriise** [comment]
I would also ask to consider moving this section further up in the README, but thats just a preference.

File: README.md [hunk 3]
```diff
@@ -72,9 +72,24 @@ By default, view nodes are strongly typed. The type of a container includes the
 
 The type erasure of View nodes is not an easy trick, as the trait has two associated types and the `rebuild` method takes the previous view as a `&Self` typed parameter. Nonetheless, it is possible. (As far as I know, Olivier Faure was the first to demonstrate this technique, in [Panoramix], but I'm happy to be further enlightened)
 
+## Prerequisites
+
+### Linux
+
+In order to run xilem on linux systems, the development library [xkbcommon] needs to be installed.
```
Comment (xStrom) [inline README.md:79]:
```suggestion
In order to run Xilem on Linux, the development library [xkbcommon] needs to be installed.
```

**Review by xStrom — APPROVED** [review]
I also think that having prerequisites documented is key. Thanks!

**Pull request merged**